key-vault-node-quickstart-v3/README.md

```diff
@@ -0,0 +1,192 @@
+```
+cd key-vault-node-quickstart-v3
+```
+```
+npm install
+```
+
+This project used 2 node modules:
+
+* [ms-rest-azure](https://www.npmjs.com/package/ms-rest-azure) 
+* [azure-keyvault](https://www.npmjs.com/package/azure-keyvault)
+
+## Publish the web application to Azure
+
+Below are the few steps we need to do
+
+- The 1st step is to create a [Azure App Service](https://azure.microsoft.com/services/app-service/) Plan. You can store multiple web apps in this plan.
+
+    ```
+    az appservice plan create --name myAppServicePlan --resource-group myResourceGroup
+    ```
+- Next we create a web app. In the following example, replace <app_name> with a globally unique app name (valid characters are a-z, 0-9, and -). The runtime is set to NODE|6.9. To see all supported runtimes, run az webapp list-runtimes
+    ```
+    # Bash
+    az webapp create --resource-group myResourceGroup --plan myAppServicePlan --name <app_name> --runtime "NODE|6.9" --deployment-local-git
+    # PowerShell
+    az --% webapp create --resource-group myResourceGroup --plan myAppServicePlan --name <app_name> --runtime "NODE|6.9"
+    ```
+    When the web app has been created, the Azure CLI shows output similar to the following example:
+    ```
+    {
+      "availabilityState": "Normal",
+      "clientAffinityEnabled": true,
+      "clientCertEnabled": false,
+      "cloningInfo": null,
+      "containerSize": 0,
+      "dailyMemoryTimeQuota": 0,
+      "defaultHostName": "<app_name>.azurewebsites.net",
+      "enabled": true,
+      "deploymentLocalGitUrl": "https://<username>@<app_name>.scm.azurewebsites.net/<app_name>.git"
+      < JSON data removed for brevity. >
+    }
+    ```
+    Browse to your newly created web app and you should see a functioning web app. Replace <app_name> with a unique app name.
+
+    ```
+    http://<app name>.azurewebsites.net
+    ```
+    The above command also creates a Git-enabled app which allows you to deploy to azure from your local git. 
+    Local git is configured with url of 'https://<username>@<app_name>.scm.azurewebsites.net/<app_name>.git'
+
+- Create a deployment user
+    After the previous command is completed you can add add an Azure remote to your local Git repository. Replace <url> with the URL of the Git remote that you got from Enable Git for your app.
+
+    ```
+    git remote add azure <url>
+    ```
+
+## Enable Managed Service Identity
+
+Azure Key Vault provides a way to securely store credentials and other keys and secrets, but your code needs to authenticate to Key Vault to retrieve them. Managed Service Identity (MSI) makes solving this problem simpler by giving Azure services an automatically managed identity in Azure Active Directory (Azure AD). You can use this identity to authenticate to any service that supports Azure AD authentication, including Key Vault, without having any credentials in your code.
+
+Run the assign-identity command to create the identity for this application:
```

Comment:
You use assign-identity here so it looks like that is the actual command, but you use "identity assign" below.  

Consider:

Run the "identity assign" command...